modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java

```diff
@@ -2019,6 +2042,21 @@ protected void validateUrlTitle(
 		}
 	}
 
+	private void _validateExternalReferenceCode(
+			String externalReferenceCode, long groupId)
+		throws PortalException {
+
+		KBArticle kbArticle = kbArticlePersistence.fetchByG_ERC(
```

Comment:
This shouldn't use fetchByG_ERC_V method? I thought that the generation with the manual ERC has two finders, one like this but that return a Collection, and the fetchByG_ERC_V that returns a KBArticle

Reply:
Forget it. I've seen is changed later in another commit